### DISCUSSION TITLE
Tools for Docker

### DISCUSSION THREAD

**Discussion by Daniellled** [microsoft/WindowsAppSDK#1845] [Q&A] [answered]
Does anyone know if a page like https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=vs-2022 existing to guide users so they can compile WindowsAppSDK applications within their CICD pipeline?

**Comment by andrewleader** [comment, marked as answer]
Hey @Daniellled, as I mentioned in another thread, we're working on adding some CI docs, in the meantime you can check out this GitHub Workflow for building a WinUI 3 app from GitHub actions: https://github.com/andrewleader/WindowsAppSDKGallery/blob/main/.github/workflows/dotnet-desktop.yml#L102
